src/main/java/com/sahaya/sahayaservices/models/GetUserName.java

```diff
@@ -1,20 +0,0 @@
-package com.sahaya.sahayaservices.models;
-
-public class GetUserName {
```

Comment:
We should never name a class or function to start with Get or Is, rename it. @angeleenajohn @anilapo @swatha @KarthikaNK @deepikaakshaya

Reply:
We are not using that model anymore,we have removed that model class